src/components/modals/AutoLogoutModal.tsx

```diff
@@ -67,92 +63,88 @@ export const AutoLogoutModal = (props: Props) => {
     bridge.resolve(undefined)
   })
 
-  // Default to `hours` if it's disabled:
+  // Default to hours if it's disabled:
   React.useEffect(() => {
     if (displayTime.value === 0) setDisplayTime({ ...displayTime, measurement: 'hours' })
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [])
 
+  const initPositionNumber = React.useMemo(
+    () => displayTime.value,
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    []
+  )
+
+  const initPositionUnit = React.useMemo(
```

Comment:
Same thing here. `const [initPositionUnit] = React.useState( () => unitLabelValueList.findIndex(option => option.value === displayTime.measurement))`. Note the use of the function here - this means we will only evaluate the `findIndex` once on first render.